Dockerfile

```diff
@@ -270,8 +270,9 @@ RUN --mount=type=bind,from=auto-gptq-cache,src=/usr/src/auto-gptq-wheel,target=/
 
 
 # Install ibm-fms from branch (required for spec. decoding)
-RUN git clone https://github.com/foundation-model-stack/foundation-model-stack.git --branch paged_attn_speculator_minimal_handoff --single-branch && \
-    cd foundation-model-stack && git checkout 14e736864b05b6b6f67fbac8fccaaaf8ac0d4739 && \
+# TODO: switch this out for fms-extras main once speculative decoding fully in
+RUN git clone https://github.com/JRosenkranz/fms-extras.git --branch speculative_decoding_e2e --single-branch && \
+    cd fms-extras && git checkout speculative_decoding_e2e && \
```

Comment:
How do you want to proceed with these two PRs? Should we merge the `fms-extras` one first, followed by this one?

Reply:
nvm - I've merged the other one now.